### DISCUSSION TITLE
Show the Views of Playlists

### DISCUSSION THREAD

**Discussion by jabbab** [sigma67/ytmusicapi#414] [Ideas]
Why wont we pull the views of a playlist while also pulling every other data? For my use case it would make sense.
Maybe something to consider.

![image](https://github.com/sigma67/ytmusicapi/assets/38084608/d172974a-d888-45a3-86bc-4f4e8147cd10)

**Comment by sigma67** [comment]
Probably something that has only been added recently to the UI. What page is this from? Library playlists?

**Reply by jabbab** [reply]
Oh didn't knew this. From the search page, under the community playlists.
https://music.youtube.com/search?q=edm

Or directly on the playlists page example:
https://music.youtube.com/playlist?list=PLw-VjHDlEOgs658kAHR_LAaILBXb-s6Q5
